app.py

```diff
@@ -101,7 +112,8 @@ def update_cache():
                 unique_buses[bus]
             )
             predictions.append(pred)
-            
+
+        MetricsHandler.stops_count.inc(1)
```

Comment:
```suggestion
        MetricsHandler.stops_count.inc()
```

Reply:
from the docs

https://prometheus.github.io/client_python/instrumenting/counter/